docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +83,155 @@ metadata:
+
+### Format
+
+Each rule consists of a semicolon separated list of plans that the rule applies to. Additionally, a plan can be extended with `:region` suffix that makes the rule evaluate to true only if a cluster is provisioned in the specified region. List entries comply with the format `<PLAN_ID_1>:<REGION_ID_1>;<PLAN_ID_2>:<REGION_ID_2>`. Either plan or region (but never both) can be specified as wildcard `*` meaning all plans or regions should apply for specific second value. The following example lists valid and invalid configuration values:
+* `trial` - valid, rule is evaluated to true for the `trial` plan in all regions,
+* `trial:*` - valid, rule is evaluated to true for the `trial` plan in all regions,
+* `trial:eu` - valid, rule is evaluated to true for the `trial` plan in the `eu` region,
+
+* `*:eu` - valid, rule is evaluated to true for all plans in the `eu` region,
+* `eu:*` - invalid, plan must be specified in the first part of `<PLAN_ID>:<REGION_ID>` pair,
+* `*:*` - invalid, at least one of the values must be specified in the `<PLAN_ID>:<REGION_ID>` pair.
+* `*:eu;trial:eu` - valid, rule is evaluated to true for all plans in the `eu` region and for the `trial` plan in the `eu` region, configuration can be duplicated
+* `trial:eu;trial:us` - valid, rule is evaluated to true for trials plans but only in `eu` and `us` regions,
+
+### Validation
+
+Rules validation is done during the KEB startup. If the configuration is invalid, KEB will not start and an error message will be displayed in the logs. The constraints used for validation include:
+* Rules format check - all the rules must comply with the format specified above.
+* Plan existence check.
+
+### Examples
+
+The section provides configuration examples of secret bindings selection rules and shows what hyperscaler pools they correspond to.
+
+By default, KEB uses the `hyperscaler-type` label to search for secrets without applying any sufixes to the label's value. It means that without any configuration the search is done by simple hyperscaler type value implemented in [KEB](https://github.com/kyma-project/kyma-environment-broker/blob/main/common/hyperscaler/hyperscaler_type.go).
+
+```
+hap: 
+  platformRegionRule: ""
+  clusterRegionRule: ""
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/default-pool.drawio.svg)
+
+---
+
+The following configuration is an example of a rule that appends the platform region to the `hyperscaler-type` label when searching for secrets for all gcp clusters. The configuration means that there would be no possibillity to configure a pool with label `hyperscalerType: gcp` since KEB would search for `hyperscalerType: gcp__<PLATFOR_REGION>` bindings only.
+
+```
+hap: 
+  platformRegionRule: "gcp"
+  clusterRegionRule: ""
+  sharedRule: ""
+  euAccessRule: ""
+```
+
+![hap-workflow](../assets/gcp-platform-pool.drawio.svg)
```

Comment:
What is the purpose of crossed out pools/rectangles? IMHO it is not needed.
If you want to specify everything that could not be reached/searched for/configured it won't be comprehensible.

Reply:
To point out what is the difference from the last example and what generic (pools with provider only) are available.